scvi/models/modules.py

```diff
@@ -266,35 +278,41 @@ def __init__(
         n_cat_list: Iterable[int] = None,
         n_layers: int = 1,
```

Comment:
Should we add in the doc the exact value that makes it linear ? If n_layers > 1 then it wouldn't be linear right ? Should we otherwise remove n_layers from this ?

Reply:
I will rename to `n_layers_encoder` because decoder is hardcoded.